js/components/portal-dashboard/question-area.tsx

```diff
@@ -48,8 +49,10 @@ export class QuestionArea extends React.PureComponent<IProps>{
         </div>
         <div className={`${css.questionTextArea} ${!useMinHeight ? css.minHeight : ""}`} data-cy="question-content">
           <div className={css.questionText}>
-            {prompt ? striptags(prompt.replace(/&nbsp;/g,' ')) : ""}
+          {prompt ? renderHTML(prompt) : ""}
+            {/* {prompt ? striptags(prompt.replace(/&nbsp;/g,' ')) : ""} */}
           </div>
+          { drawingPrompt && this.renderImageQuestionPrompt(drawingPrompt)}
```

Comment:
Tab in for proper alignment.

Reply:
Looking more closely at the story spec, the prompt goes second.  This should be `{ prompt && this.renderImageQuestionPrompt(prompt) }`